### PR TITLE
feat: add task list page route and placeholder [DET-3220]

### PR DESCRIPTION
**Pull request by hamidzr**
## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/routes/index.ts [hunk 1]
```diff
@@ -33,8 +34,19 @@ const dashboardRoute =
     title: 'Dashboard',
   };
 
+const TaskListRoute =
```
Comment (hkang1) [inline webui/react/src/routes/index.ts:37]: nit: `taskListRoute`

File: webui/react/src/routes/index.ts [hunk 2]
```diff
@@ -60,6 +72,7 @@ export const sidebarRoutes: RouteConfigItem[] = [
     path: '/ui/experiments',
     title: 'Experiments',
   },
+  // TaskListRoute,
```
Comment (hkang1) [inline webui/react/src/routes/index.ts:75]: chore: drop this line

Comment (hamidzr) [reply webui/react/src/routes/index.ts:75]: hmm was trying to show we'd flip that once we are ready to show the page, but that's simple enough

Comment (hamidzr) [reply webui/react/src/routes/index.ts]: removed it

**Pull request merged**